Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.py

```diff
@@ -214,8 +214,7 @@ def getEventsByQuery(session, queryData, max_results, extended_data, max_wait_ti
             if len(key["data"]) == 0 or key["data"][0] == "No report results found.":
                 md = "No report results found."
                 break
-            else:
```

Comment:
is this `ruff`'s change, or ours?

Reply:
[ruff](https://docs.astral.sh/ruff/rules/superfluous-else-break/)
But I fixed it manually.

Reply:
done